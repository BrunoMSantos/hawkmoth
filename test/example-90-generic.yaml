directive-arguments:
- example-90-generic.c
