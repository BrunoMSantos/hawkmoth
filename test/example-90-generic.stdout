
Source files may contain documentation comments not attached to any C
constructs. They will be included as generic documentation comments, for
example to describe the design of :c:type:`list` frobnication using
:c:func:`frob`.


